master/reference/node/configuration.md

```diff
@@ -144,3 +144,42 @@ e.g.
 IP_AUTODETECTION_METHOD=interface=eth.*
 IP6_AUTODETECTION_METHOD=interface=eth.*
 ```
+
+### Node readiness
+
+**Felix**
```

Comment:
Now that we've got CLI flags we can pass, I think we can condense this down to simply explain what the different flags are - `bird, bird6, felix`